cmd/seid/cmd/root.go

```diff
@@ -374,6 +376,8 @@ func initAppConfig() (string, interface{}) {
 		EVM evmrpc.Config `mapstructure:"evm"`
 
 		ETHReplay replay.Config `mapstructure:"eth_replay"`
+
+		ETHBlockTest blocktest.Config `mapstructure:"eth_block_test"`
```

Comment:
need to add this here https://github.com/sei-protocol/sei-chain/pull/1455/files#diff-846ffb3902524a4be27690778841ca410dbb2eabdd5387704158e2e8779f04c6R421 and here https://github.com/sei-protocol/sei-chain/pull/1455/files#diff-846ffb3902524a4be27690778841ca410dbb2eabdd5387704158e2e8779f04c6R494